include/rocksdb/statistics.h

```diff
@@ -447,6 +447,9 @@ struct HistogramData {
   double min = 0.0;
 };
 
+// Reduce statistics overhead by disabling collection of certain types of stats
```

Comment:
Maybe mention `StatsLevel` explicitly. For example, StatsLevel controls what statistics to record...